vue/index.html

```diff
@@ -6,6 +6,6 @@
     </head>
     <body>
         <div id="app"></div>
-        <script src="dist/build.js"></script>
+        <script src="/js/build.js"></script>
```

Comment:
@p0psicles did you end up adding `/vue/dist/build.js` to the static resources?

It may be better to do that than having the build use `static/js/` as it'll allow the vue dir to be completely separate from Medusa's code this allows the dir itself to be used with nginx, apache, caddy, etc. as a static site.